python_modules/dagster-graphql/dagster_graphql/schema/backfill.py

```diff
@@ -288,7 +311,12 @@ class Meta:
     assetSelection = graphene.List(graphene.NonNull(GrapheneAssetKey))
     partitionSetName = graphene.Field(graphene.String)
     timestamp = graphene.NonNull(graphene.Float)
+    creationTime = graphene.NonNull(
+        graphene.Float
+    )  # for RunsFeedEntry interface - dupe of timestamp
+    startTime = graphene.Float()  # for RunsFeedEntry interface - dupe of timestamp
```

Comment:
nit: maybe we should consolidate on to `timestamp` instead since its a `float` ? I feel like `time` is a better name for returning like a datetime object

Reply:
i like that

Reply:
my thinking with keeping createTime, startTime, endTime was that in the future where everything can be a single run, we won't have these duplicate fields on GrapheneRun